client/verta/verta/deployment/update/rules.py

```diff
@@ -32,39 +32,56 @@ def _from_dict(rule_dict):
         rule_name = rule_dict['rule_parameters'][0]['name']
         rule_value = float(rule_dict['rule_parameters'][0]['value'])
 
-        RULE_SUBCLASSES = [AverageLatencyThresholdRule, P90LatencyThresholdRule, ErrorRateRule]
+        RULE_SUBCLASSES = [
+            MaximumAverageLatencyThresholdRule, MaximumP90LatencyThresholdRule, MaximumRequestErrorPercentageThresholdRule, MaximumServerErrorPercentageThresholdRule
+        ]
```

Comment:
That's a lot of characters 😆 
```suggestion
        RULE_SUBCLASSES = [
            MaximumAverageLatencyThresholdRule,
            MaximumP90LatencyThresholdRule,
            MaximumRequestErrorPercentageThresholdRule,
            MaximumServerErrorPercentageThresholdRule,
        ]
```

Reply:
Fixed.